lms/static/js/dashboard/legacy.js

```diff
@@ -126,9 +126,17 @@
         });
 
         $(".action-unenroll").click(function(event) {
+            var track_info = $(event.target).data("track-info");
```

Comment:
It's probably cheap to build a jQuery collection from a DOM object, but since `$(event.target)` is used 6 times, it might be worthwhile caching it in a variable anyway.